contribs/gmf/examples/style/mobileapp-map.less

```diff
@@ -7,3 +18,162 @@ gmf-map {
     left: 0;
   }
 }
+
+.ol-zoom {
+  background-color: transparent;
+  position: absolute;
+  right: 0;
+  left: auto;
+  top: 52px;
+  margin: 10px;
+  button {
+    height: @map-tools-size;
+    width: @map-tools-size;
+    background-color: @map-tools-bg-color;
+    border: solid 1px @map-tools-color;
+    color: @map-tools-color;
+    padding: 0;
+    margin-bottom: 3px;
+  }
+}
+
+.ol-attribution {
+  display:none;
+}
+
+.ol-scale-line, .ol-scale-line-inner {
+  background-color: @map-tools-bg-color;
+  border-color: @map-tools-color;
+  border-width: 2px;
+  color: @map-tools-color;
+}
+
+.geolocation {
+  background-color: @map-tools-bg-color;
+  border: solid 1px @map-tools-color;
+  height: @map-tools-size;
+  width: @map-tools-size;
+  position: absolute;
+  right: 0;
+  left: auto;
+  top: 137px;
+  margin-right: 13px;
+  z-index: @above-content-index;
+  &:after {
+    content: 'b';
+    font-family: gmf-icons;
+    color: @map-tools-color;
+    font-size: @map-tools-size;
+    margin-left: 4px;
+  }
+}
+
+
+.search {
+  position: absolute;
+  top: 13px;
+  left: 0;
+  right: 0;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 6px 10px;
+  background-color: @map-tools-bg-color;
+  width: 162px + @map-tools-size + @map-tools-size;
+  border-radius: 25px;
+  border: solid 1px @map-tools-color; 
+  z-index: @content-index;
+  input {
+    width: 150px;
+    margin: 0 10px;
+    padding: 0 5px;
+    border-top: none;
+    border-bottom: none;
+    border-radius: 0;
+  }
+}
+
+.nav-trigger {
+  position: absolute;
+  top: 13px;
+  left: 50vw;
+  margin-top: 2px;
+  padding: 0;
+  background-color: transparent;
+  border: none;
+  height: @map-tools-size;
+  width: @map-tools-size;
+  z-index: @above-content-index;
+  &:after {
+    font-family: gmf-icons;
+    color: @map-tools-color;
+    font-size: @map-tools-size;
+    margin-left: 4px;
+  }
+}
+
+.nav-left-trigger {
+  margin-left: -102px;
+  &:after {
+    content: 'a';
+  }
+}
+
+.nav-right-trigger {
+  margin-left: 77px;
+  &:after {
+    content: 'd';
+  }
+}
+
+//For tablet only
+@media (min-width: @screen-xs-max) {
```

Comment:
You shouldn't mix min/max when using media queries & variables (use min-width with min values).
@screen-xs-max: 767px
@screen-sm-min: 768px
See https://github.com/twbs/bootstrap/blob/master/less/variables.less#L314-L317